src/sys/socket/sockopt.rs

```diff
@@ -478,6 +492,53 @@ unsafe impl<'a> Set<'a, usize> for SetUsize {
     }
 }
 
+/// Getter for a `OsString` value.
+struct GetOsString<T: AsRef<[u8]>> {
```

Comment:
This should be `AsMut`, not `AsRef`, since `libc::getsockopt` is going to modify it.

Reply:
Done